docs/src/man/basics.md

```diff
@@ -0,0 +1,2042 @@
+      │ Int64  Int64  Int64  String   String           String            Int64 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0     67      2  own      NA               little                  ⋯
+    2 │     1     22      2  own      little           moderate
+    3 │     2     49      1  own      little           NA
+    4 │     3     45      2  free     little           little
+    5 │     4     53      2  free     little           little                  ⋯
+    6 │     5     35      1  free     NA               NA
+    7 │     6     53      2  own      quite rich       NA
+    8 │     7     35      3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮            ⋮                ⋮                ⋱
+  994 │   993     30      3  own      little           little                  ⋯
+  995 │   994     50      2  own      NA               NA
+  996 │   995     31      1  own      little           NA
+  997 │   996     40      3  own      little           little
+  998 │   997     38      2  own      little           NA                      ⋯
+  999 │   998     23      2  free     little           little
+ 1000 │   999     27      2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+In the given code of block `:` means that the columns will get copied:
+```jldoctest dataframe
+julia> german[:, Not(2)]
+1000×9 DataFrame
+  Row │ id     Sex     Job    Housing  Saving accounts  Checking account  Cred ⋯
+      │ Int64  String  Int64  String   String           String            Int6 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0  male        2  own      NA               little                 ⋯
+    2 │     1  female      2  own      little           moderate
+    3 │     2  male        1  own      little           NA
+    4 │     3  male        2  free     little           little
+    5 │     4  male        2  free     little           little                 ⋯
+    6 │     5  male        1  free     NA               NA
+    7 │     6  male        2  own      quite rich       NA
+    8 │     7  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮       ⋮       ⋮            ⋮                ⋮               ⋱
+  994 │   993  male        3  own      little           little                 ⋯
+  995 │   994  male        2  own      NA               NA
+  996 │   995  female      1  own      little           NA
+  997 │   996  male        3  own      little           little
+  998 │   997  male        2  own      little           NA                     ⋯
+  999 │   998  male        2  free     little           little
+ 1000 │   999  male        2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+## Views
```

Comment:
you have already introduced views above - consider reordering the content.

Reply:
I had taken views because I was discussing `setindex!` and I wanted to talk about SubDataFrame as well. I needed view to create a SubDataFrame. But I will describe why we have used it and what it is.